### PR TITLE
Added a new folder List-currencies and file List-currencies

### PR DESCRIPTION
**Pull request by MathewWith**
Положил папку List-currencies  в helpers потому что не хотел создавать доп папку components ради одной List-currencies, нормально?

### CODE REVIEW

**Comment by CodingRuslan** [comment]
No, it's not normal. You put component into helpers it's not right. Read about helper functions

File: src/App.tsx [hunk 1]
```diff
@@ -11,7 +12,8 @@ function App() {
 
   return (
     <div className="App">
-      
+      <h1>Change money</h1>
+      {ListCurrencies(allCurrencies)}
```
Comment (CodingRuslan) [inline src/App.tsx]: You use components like function it;s not right. 
Why we have a props ?

File: src/helpers/list-currencies/List-currencies.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import {useState} from "react"
+
+export const ListCurrencies = (allCurrencies: string[]) => {
+
+    const [currencyOne, setCurrencyOne] = useState(allCurrencies[0])
```
Comment (CodingRuslan) [inline src/helpers/list-currencies/List-currencies.tsx]: CurrencyLeft and CurrencyRight - better name

File: src/helpers/list-currencies/List-currencies.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    const [currencyTwo, setCurrencyTwo] = useState(allCurrencies[0])
+
+    return (
+        <div className="listCurrencies">
```
Comment (CodingRuslan) [inline src/helpers/list-currencies/List-currencies.tsx]: think about right className

File: src/helpers/list-currencies/List-currencies.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            <select onChange={(e) => setCurrencyOne(e.target.value)}>
+                {
+                    allCurrencies.map((cur) => {
+                       return <option key={cur}>{cur}</option>
+                    })
+                }
+            </select>
+            <select onChange={(e) => setCurrencyTwo(e.target.value)}>
```
Comment (CodingRuslan) [inline src/helpers/list-currencies/List-currencies.tsx]: I see code repeat. May be better to create Select component and reuse it ?
Read DRY principle

File: src/App.tsx (hunk 1 above)
Comment (MathewWith) [reply src/App.tsx]: Yes, this is my mistake. Initially I used func(list-currencies) but I needed to use useState and I couldn't do it in func. And then I forgot, yes

**Comment by CodingRuslan** [comment]
You have conflicts in PRs, fix it

**Pull request merged**